src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -970,6 +1004,43 @@ def has_args(call_node: ast.Call, required_arg_ids: Tuple[str, ...]) -> bool:
     return False
 
 
+def merge_entry_points(
+    *entry_points: Tuple[str, Dict[str, Dict[str, str]]]
```

Comment:
I renamed the arg, but kept the type hint, as it comes in as `(desc, entry_points), (desc, entry_points), ...`
it would complicate the code calling this method if we would want to swap the members of the tuples...